clang/lib/Interpreter/DeviceOffload.cpp

```diff
@@ -50,25 +50,6 @@ IncrementalCUDADeviceParser::Parse(llvm::StringRef Input) {
   if (!PTU)
     return PTU.takeError();
 
-  auto PTX = GeneratePTX();
```

Comment:
If this code moved, can't we rely on the base class implementation of `Parse` here?